vendors/ti/boards/cc3220_launchpad/aws_tests/config_files/ota_config.h

```diff
@@ -0,0 +1,195 @@
+ * the request message after being idle for this amount of time.
+ */
+#define otaconfigFILE_REQUEST_WAIT_MS           10000U
+
+/**
+ * @brief The maximum allowed length of the thing name used by the OTA agent.
+ *
+ * AWS IoT requires Thing names to be unique for each device that connects to the broker.
+ * Likewise, the OTA agent requires the developer to construct and pass in the Thing name when
+ * initializing the OTA agent. The agent uses this size to allocate static storage for the
+ * Thing name used in all OTA base topics. Namely $aws/things/<thingName>
+ */
+#define otaconfigMAX_THINGNAME_LEN              64U
+
+/**
+ * @brief The maximum number of data blocks requested per request from the OTA
+ * streaming service.
+ *
+ * This configuration parameter is sent with data requests and represents the
+ * maximum number of data blocks the service will send in a single response.
+ * The amount of data requested and therefore received in a single response
+ * (in bytes) is equal to:
+ *
+ * (otaconfigMAX_NUM_BLOCKS_REQUEST * ( pow(2,otaconfigLOG2_FILE_BLOCK_SIZE) )
+ *
+ * @note If the device has a incoming TLS I/O buffer set to be less than the
+ * standard default size of 16KB, then the amount of bytes being requested has
+ * to be less than the the size of the incoming TLS I/O buffer.
+ * <br><br>
+ *
+ * For example, if the device has otaconfigLOG2_FILE_BLOCK_SIZE set to '12',
+ * then each data block will be 4096 bytes large. If the device also has a
+ * incoming TLS I/O bufferthat is 8KB large, then the
+ * otaconfigMAX_NUM_BLOCKS_REQUEST must be set to '1' so that the data received
+ * in one response doesn't surpass the size of the incoming TLS I/O buffer.
+ *
+ * @note If the device has a incoming TLS I/O buffer set to be equal to the
+ * standard default size of 16Kb, then the maximum limit for this must be
+ * calculated from the maximum data response limit (128 KB from service)
+ * divided by the block size. For example if block size is set as 1 KB then the
+ * maximum number of data blocks that we can request is 128/1 = 128 blocks.
+ * Configure this parameter to this maximum limit or lower based on how many
+ * data blocks response is expected for each data requests.
+ *
+ * @note This must be set to a value larger than zero.
+ */
+#define otaconfigMAX_NUM_BLOCKS_REQUEST         4U
+
+/**
+ * @brief The maximum number of requests allowed to send without a response before we abort.
+ *
+ * This configuration parameter sets the maximum number of times the requests are made over
+ * the selected communication channel before aborting and returning error.
+ *
+ */
+#define otaconfigMAX_NUM_REQUEST_MOMENTUM       32U
+
+/**
+ * @brief The number of data buffers reserved by the OTA agent.
+ *
+ * This configurations parameter sets the maximum number of static data buffers used by
+ * the OTA agent for job and file data blocks received.
+ */
+#define otaconfigMAX_NUM_OTA_DATA_BUFFERS       4U
```

Comment:
Number of blocks requested plus 1

Reply:
Updated in https://github.com/aws/amazon-freertos/pull/3179/commits/1a91157fe268d8068568eebd9fcd04a98e7cd81a